### PR TITLE
Add known issues section for Kiali

### PR DESCRIPTION
**Pull request by lilitgh**
**Description**

Changes proposed in this pull request:

- Add known issues section for Kiali

### CODE REVIEW

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md [hunk 1]
```diff
@@ -32,6 +32,8 @@ See the overview of all changes in this release:
 - [Observability](#observability) - Early version of Kiali added to Istio
 - [Service Mesh](#service-mesh) - Istio update to version 1.1.6
 
+Read about a known issue for [Observability](###known-issues).
```
Comment (magicmatatjahu) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]: I think that triple `#` us unnecessary and link doesn't work :) 
```markdown
Read about a known issue for [Observability](#known-issues).
```

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md [hunk 2]
```diff
@@ -123,6 +125,18 @@ We prepared a self-contained example that shows how to configure an Event trigge
...
 
+### Known issues
+
+Kiali is not accessible after installation when enabled as a part of the installer configuration. As an example, for minikube it should be accessible at https://kiali.kyma.local.
+As a workaround prefix the hosts attribute in kiali-virtualservice resource with "kiali":
+```
+kubectl -n istio-system edit virtualservices kiali-virtualservice
+```
+```
+  hosts:
```
Comment (magicmatatjahu) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]: Please remove white spaces

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md [hunk 3]
```diff
@@ -123,6 +125,18 @@ We prepared a self-contained example that shows how to configure an Event trigge
...
+  - kiali.kyma.local
```
Comment (magicmatatjahu) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]: Please remove white spaces

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md (hunk 1 above)
Comment (lilitgh) [reply content/blog-posts/2019-06-13-release-notes-1.2/index.md]: changed

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md [hunk 4]
```diff
@@ -123,6 +125,18 @@ We prepared a self-contained example that shows how to configure an Event trigge
 
 Early integration of Kiali is available as part of Istio. To enable Kiali, ensure that the monitoring module is installed and set the **kiali.enabled** parameter to `true`. The Kiali UI will be accessible under the `kiali` subdomain. The early integration is based on static user security. To learn how to get the Kiali UI password, see [this](https://github.com/kyma-project/kyma/tree/master/resources/istio) document.
 
+### Known issues
+
+Kiali is not accessible after installation when enabled as a part of the installer configuration. As an example, for minikube it should be accessible at https://kiali.kyma.local.
```
Comment (kazydek) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]:
```suggestion
Kiali is not accessible after installation when enabled as part of the Installer configuration. It should be accessible on Minikube at `https://kiali.kyma.local`.
```

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md [hunk 5]
```diff
@@ -123,6 +125,18 @@ We prepared a self-contained example that shows how to configure an Event trigge
...
+As a workaround prefix the hosts attribute in kiali-virtualservice resource with "kiali":
```
Comment (kazydek) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]:
```suggestion
To access it this way, use the workaround for now and add the `kiali` prefix to the **hosts** attribute in the `kiali-virtualservice` resource:
```

File: content/blog-posts/2019-06-13-release-notes-1.2/index.md (hunk 1 above)
Comment (kazydek) [inline content/blog-posts/2019-06-13-release-notes-1.2/index.md]:
```suggestion
Read about a known issue for [Observability](#known-issues).
```

**Pull request merged**